bin/cli.js

```diff
@@ -13,11 +13,15 @@ var pessimist = require('pessimist')
     .alias('c', 'concurrency')
     .default('c', 5)
     .describe('c', 'number of simultaneous fetches (default: 5)')
+    .alias('q', 'query')
```

Comment:
Boolean parameters are usually specifiied as either being present or not so in this case not having it in the command line would be `no` and the parameter's mere existence would be `yes`. You can do this in the `optimist`library with:
```
    .boolean('q')
    .describe('q', 'remove query parameters')
```